Tasks/Common/azure-arm-rest/azure-arm-common.ts

```diff
@@ -49,11 +55,20 @@ export class ApplicationTokenCredentials {
         this.authorityUrl = authorityUrl;
         this.activeDirectoryResourceId = activeDirectoryResourceId;
         this.isAzureStackEnvironment = isAzureStackEnvironment;
+        this.scheme = AzureModels.Scheme[scheme];
+        this.msiPort = msiPort ? msiPort : '50342';
```

Comment:
This port will no longer be required. Please check the revised mail on this regard.